features/desktop/search.feature

```diff
@@ -0,0 +1,31 @@
+@javascript
+Feature: search for users and hashtags
+  In order to find my friends on diaspora
+  As a user
+  I want search for them
+
+Background:
+  Given following users exist:
+  | username | email |
+  | Bob Jones | bob@bob.bob |
+  | Alice Smith | alice@alice.alice |
+  And I sign in as "bob@bob.bob"
+
+Scenario: search for a user and go to its profile
+  When I enter "Alice Sm" in the search input
+  Then I should see "Alice Smith" within ".ac_results"
+
+  When I click on the first search result
+  Then I should see "Alice Smith" within ".profile_header #name"
+
+  When I enter "Trinity" in the search input
+  Then I should see "Search for Trinity" within ".ac_even"
+
+  When I click on the first search result
+  Then I should see "Users matching TrinityLooking for posts tagged #Trinity?" within "#search_title"
```

Comment:
`Then I should see "Users matching Trinity" within "#search_title"`
should be sufficient.